ddl/ddl.go

```diff
@@ -400,6 +402,8 @@ const (
 	codeWrongTableName        = 1103
 	codeBlobKeyWithoutLength  = 1170
 	codeInvalidOnUpdate       = 1294
+	codeUnknownTypeLength     = 1300
+	codeUnknownFractionLength = 1301
```

Comment:
Are there MySQL error codes? If there aren't MySQL error codes, move them to line 380 and error codes are numbered starting from 9.